src/assets/styles/variables.scss

```diff
@@ -49,3 +49,6 @@ $containerPaddingBottom: 4rem;
 // Line height
 $baseLineHeight: 1.2;
 $largeLineHeight: 1.6;
+
+// Maxwidth for text paragraphs
+$textParagraphMaxWidth: 50rem;
```

Comment:
https://i.dlpng.com/static/png/5480524_thumb.png
how come you come up with this idea??

Reply:
Anniina. It is recommended max-line-length for text.